prime-router/src/main/kotlin/azure/DatabaseLookupTableAccess.kt

```diff
@@ -154,13 +154,14 @@ class DatabaseLookupTableAccess(private val db: DatabaseAccess = DatabaseAccess(
      * Create a new table [version] for a [tableName] using the provided [tableData].
```

Comment:
Update the function description to include the new parameter.

Reply:
Done.  Thanks